types/validator/lib/blacklist.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module "validator/lib/blacklist" {
```

Comment:
Is the module declaration redundant now that the export is in a file with that path?

Reply:
Let me give it a shot!

Reply:
nope that wasn't happy lol

Reply:
says you can’t declare variables and exports outside of a declare block